eventbrite/britecharts
# Adding support of negative values for line chart

## Description


Following the advice in #338, I've made the modification in `line.js` file. In addition, I've introduced `horizontal-grid-line--highlighted` CSS class that highlights the 0-axis in case there are negative values in the chart. Also I've changed how styles are imported in sandbox (see sandbox/index.html and sandbox/sandbox.js changes). Personally I don't care too much about them but I found it's easier to use sandbox if styles are re-compiled dynamically. Let me know if you want to exclude this change completely or if you want me to create another PR for this.  

## Motivation and Context


Fixed this issue: https://github.com/eventbrite/britecharts/issues/338

## How Has This Been Tested?



I've created three tests.
1. Check that the first Y-tick is negative if there are negative values.
2. Check that the 0-axis has `horizontal-grid-line--highlighted` class if there are negative values.
3. Check that the 0-axis has NO `horizontal-grid-line--highlighted` class if there is NO negative values.

## Screenshots:

![Screenshot 2020-02-21 at 14 21 23](https://user-images.githubusercontent.com/6004013/75037913-82e3c380-54b5-11ea-8fa2-e41e2d20af1e.png)

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)

## Checklist:


- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] Code follows the [API Guidelines](http://eventbrite.github.io/britecharts/topics-index.html#toc5__anchor)
- [ ] Added tests to cover changes
